templates/zerver/emails/missed_message.source.html

```diff
@@ -18,7 +18,7 @@
 <div class="email-preferences">
     &mdash;<br>
     {% if mention %}
-    You are receiving this because you were mentioned.<br>
+    You are receiving this because you were mentioned in {{ realm_name }}.<br>
```

Comment:
Do we want to make the realm name a link?  I can see arguments either way -- it's distracting as a second link other than the one you probably want to click that goes directly to the message vs. it makes it clear that the server is a thing with a URL.

Reply:
yeah, I thought about that and then decided against it due to the existence of the "View it in Zulip" link.

Reply:
OK.  We can see if anyone complains.